api/tests/unit/features/test_unit_features_views.py

```diff
@@ -529,9 +530,11 @@ def test_should_create_tags_when_feature_created(client, project, tag_one, tag_t
     "client",
     [lazy_fixture("admin_master_api_key_client"), lazy_fixture("admin_client")],
 )
-def test_add_owners_adds_owner(client, project):
+def test_add_owners_adds_owner_when_users_not_connected(client, project):
```

Comment:
I think this is the negative case, right? So the test name should be more like: 

```suggestion
def test_add_owners_fails_if_user_not_found(client, project):
```